chapters/en/chapter5/evaluation.mdx

```diff
@@ -0,0 +1,387 @@
+# Evaluation metrics for ASR
+
+If you're familiar with the [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) from NLP, the
+metrics for assessing speech recognition systems will be familiar! Don't worry if you're not, we'll go through the
+explanations start-to-finish to make sure you know the different metrics and understand what they mean.
+
+When assessing speech recognition systems, we compare the system's predictions to the target text transcriptions,
+annotating any errors that are present. We categorise these errors into one of three categories:
+1. Substitutions (S): where we transcribe the **wrong word** in our prediction ("sit" instead of "sat")
+2. Insertions (I): where we add an **extra word** in our prediction
+3. Deletions (D): where we **remove a word** in our prediction
+
+These error categories are the same for all speech recognition metrics. What differs is the level at which we compute
+these errors: we can either compute them on the _word level_ or on the _character level_.
+
+We'll use a running example for each of the metric definitions. Here, we have a _ground truth_ or _reference_ text sequence:
+
+```python
+reference = "the cat sat on the mat"
+```
+
+And a predicted sequence from the speech recognition system that we're trying to assess:
+
+```python
+prediction = "the cat sit on the"
+```
+
+We can see that the prediction is pretty close! But some words are not quite right. We'll evaluate this prediction
+against the reference for the three most popular speech recognition metrics and see what sort of numbers we get for each.
+
+## Word Error Rate
+The _word error rate (WER)_ metric is the 'de facto' metric for speech recognition. It calculates substitutions,
+insertions and deletions on the _word level_. This means errors are annotated on a word-by-word basis. Take our example:
+
+
+| Reference:  | the | cat | sat     | on  | the | mat |
+|-------------|-----|-----|---------|-----|-----|-----|
+| Prediction: | the | cat | **sit** | on  | the |     |  |
+| Label:      | ✅   | ✅   | S       | ✅   | ✅   | D   |
+
+Here, we have:
+* 1 substitution ("sit" instead of "sat")
+* 0 insertions
+* 1 deletion ("mat" is missing)
+
+This gives 2 errors in total. To get our error rate, we divide the number of errors by the total number of words in our
+reference (N), which for this example is 6:
+
+\begin{aligned}
```

Comment:
The formula doesn't render.

Reply:
Resolved in https://github.com/huggingface/audio-transformers-course/pull/70/commits/2d0ab9194c416f9ea4b0e78e68311e9b02332e65